torchreid/apis/training.py

```diff
@@ -32,11 +32,11 @@
 def run_lr_finder(cfg, datamanager, model, optimizer, scheduler, classes,
                   rebuild_model=True, gpu_num=1, split_models=False):
     if rebuild_model:
-        tmp_model = model
+        backup_model = model
```

Comment:
Do we need backup_model in the case when we rebuild the model?

Reply:
`backup_model` exists in all the branches for safety, but indeed there's a more gentle way of using it.